app/components/widgets/EventList.vue.ts

```diff
@@ -12,11 +12,11 @@ import {
   IPlatformAuth,
   TPlatform,
   IPlatformService
-} from '../../services/platforms';
-import { CustomizationService } from '../../services/customization';
+} from '../../services/platforms/index';
+import { CustomizationService } from '../../services/customization/index';
 import Raven from 'raven-js';
-import { AppService } from 'services/app';
-import { SceneCollectionsService } from 'services/scene-collections';
+import { AppService } from 'services/app/index';
+import { SceneCollectionsService } from 'services/scene-collections/index';
```

Comment:
see above

Reply:
I personally wish to delete these files. It's not a good idea to store unused components in the repo.

Reply:
@holiber i agree with that